lib/mix/lib/mix/project.ex

```diff
@@ -378,19 +378,53 @@ defmodule Mix.Project do
   @doc """
   Returns the full path of all dependencies as a map.
 
+  ## Options
+
+    * `:depth` - only return deps to the depth level, a depth of 1
```

Comment:
Minor nitpick: what do you think about using "dependencies" instead of "deps" in the documentation?

> * `:depth` - only return dependencies to the depth level, a depth of 1 will only return top-level dependencies

The "deps" shortcut is never used in the documentation of this module :)